invenio_jobs/assets/semantic-ui/js/invenio_jobs/administration/JobSearchResultItemLayout.js

```diff
@@ -101,31 +101,24 @@ class SearchResultItemComponent extends Component {
               />
             </>
           ) : (
-            "−"
+            "-"
```

Comment:
was there something wrong 👀

Reply:
Yes, it was some weird char instead of `-`

Reply:
It was intentionally chosen as normal n-dashes look too short: -

I prefer an em-dash: —

But went with one in between to be a bit fancy: −

Unless there is a problem with the character not displaying I don't see why not choosing something that looks better